chromium_src/chrome/browser/flags/android/chrome_feature_list.cc

```diff
@@ -60,6 +60,7 @@ namespace android {
 OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kAddToHomescreenIPH, base::FEATURE_DISABLED_BY_DEFAULT},
     {kAndroidHub, base::FEATURE_DISABLED_BY_DEFAULT},
+    {tab_groups::kAndroidTabGroupStableIds, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
is there a follow-up issue for this? This feature will likely go away eventually and we should get ahead of that

Reply:
Good point, I'll create one

Reply:
Created issue https://github.com/brave/brave-browser/issues/39218